selfdrive/car/mazda/interface.py

```diff
@@ -21,7 +21,7 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=None):
     ret.carName = "mazda"
     ret.safetyModel = car.CarParams.SafetyModel.mazda
 
-    ret.dashcamOnly = True
+    ret.dashcamOnly = candidate in STEER_LOCK_CAR
```

Comment:
let's also set the community feature flag for all models since the CX9 will need to be behind it anyway